quadrat/theme.json

```diff
@@ -217,6 +217,31 @@
 				},
 				"typography": {
 					"fontSize": "var(--wp--preset--font-size--normal)"
+				},
+				"checkbox": {
```

Comment:
Are we defining these only on the child themes? these should come from Blockbase and then rebuilt into the children

Reply:
The reason I skipped Blockbase (and didn't move the other checkbox styles) is because it looks like Blockbase is using a native checkbox:

![native checkbox](https://user-images.githubusercontent.com/1645628/129063930-47e525c0-dd3a-4208-8d0b-122ede61cf90.png)

This is what it looks like when I move all these custom checkbox related styles into Blockbase:

![custom checkbox](https://user-images.githubusercontent.com/1645628/129063988-d8a859cd-046d-4208-9c1e-4072420e4cc2.png)

If we ideally want all Blockbase and children using a custom checkbox, should I make the border of the custom Blockbase one any darker, similar to the native one? My screenshots are from Firefox. Mainly asking as the box is so tiny and I'm not sure the light grey is great for accessibility. It's using the same border colour as the other form inputs too, so then I don't know if they should all change or just the checkbox border. What do you think?

Reply:
Yeah, that makes sense. Blockbase styles are based off of Blank Canvas', which looks the same on Firefox (Chrome shows a very small checkbox with a tiny border). In turn Blank Canvas is a child of Seedlet:

Seedlet | Blank Canvas
--- | ---
<img width="756" alt="Screenshot 2021-08-11 at 18 34 11" src="https://user-images.githubusercontent.com/3593343/129068152-5e68250d-7098-47fe-a975-7766b1fb3005.png"> | <img width="814" alt="Screenshot 2021-08-11 at 18 33 46" src="https://user-images.githubusercontent.com/3593343/129068160-76e2793b-37f2-4ff9-a88f-105eac8a1070.png">

I think this is a minor change and I'd rather remove as much of the CSS from the children as possible. We could make the custom checkbox on Blockbase look the same color as it does on Firefox without being the native one but using our theme.json settings, or maybe we could change all the inputs borders on Blockbase a more contrasting color. @kjellr what do you think about this?

Reply:
I'm not sure I follow. 😅 What are the two options here?

Reply:
so right now Blockbase (and Blank Canvas and Seedlet) have native checkboxes for the comments form. The idea is that since Quadrat and Skatepark used a styled one, have that code be in Blockbase too, and make it custom styled instead of native, but that brings the a11y issues @mikachan pointed out. I don't have a preference as to how to prevent that, besides wanting to avoid the code duplication of having the child themes all provide the custom css for the checkboxes instead of providing it from Blockbase.

Reply:
Thanks @kjellr, that makes sense, I'll do that! Thanks @MaggieCabrera for clarifying.